### DISCUSSION TITLE
lolminer 1.43 dual mining Crash

### DISCUSSION THREAD

**Discussion by pabloguns** [Lolliedieb/lolMiner-releases#1077] [General]
![toncoinerror](https://user-images.githubusercontent.com/99106102/152665732-da57140d-1c36-470c-828b-503ccc46370e.jpg)

Hello, could someone help me please. It kept running for a few hours and then it gave me this error.
The rig is made up of 2 rtx3060v1 + 2 6700xt. Thanks a lot

**Comment by nguyenhathienphu** [comment]
same problem.


Connecting to pool... 
Ton pool connection mode: https job polling (ton-miner compatible) (--ton-mode 1) 
DNS over HTTPS resolve failed - switching to standard resolve 
DNS over HTTPS resolve succeeded 
Connected to ethash.poolbinance.com(76.223.62.235):443  (TLS disabled)
Set Ethash stratum mode: Ethereum Proxy
Authorized worker: xxxxx 
New job received: 0xec740e Epoch: 471 Difficulty: 8.59G 
New job received: 0xec740e Epoch: 471 Difficulty: 8.59G 
https://server1.whalestonpool.com: Address is valid
https://server1.whalestonpool.com: Address is registered
Start Mining... 
New job received: 0x6b0e47 Epoch: 471 Difficulty: 8.59G 
Ton job with difficulty 274.88G and giver Ef-kkdY_B7p-77TLn2hUhM6QidWrrsl8FYWCIvBMpZKprKDH received.
------------------------------------------------------- 
          Generated light cache for epoch 471 
       Took 2356 ms, size of new DAG: 4791 MByte 
------------------------------------------------------- 
GPU 0: Starting DAG generation (normal mode)
GPU 1: Starting DAG generation (normal mode)
GPU 0: DAG build complete 16%
New job received: 0xff410a Epoch: 471 Difficulty: 8.59G 
New job received: 0xf01fb8 Epoch: 471 Difficulty: 8.59G 
New job received: 0xd1dc9d Epoch: 471 Difficulty: 8.59G 
GPU 1: DAG build complete 16%
GPU 0: DAG build complete 32%
Ton job with difficulty 274.88G and giver Ef_NSzfDJI1A3rOM0GQm7xsoUXHTgmdhN5-OrGD8uwL2JHBa received.
GPU 0: DAG build complete 48%
GPU 1: DAG build complete 32%
GPU 0: DAG build complete 64%
Ton pool connection mode: https job polling (ton-miner compatible) (--ton-mode 1) 
DNS over HTTPS resolve succeeded 
New job received: 0x80833d Epoch: 471 Difficulty: 8.59G 
GPU 0: DAG build complete 80%
GPU 1: DAG build complete 48%
New job received: 0x48fb7d Epoch: 471 Difficulty: 8.59G 
GPU 0: DAG build complete 96%
New job received: 0x9a3d71 Epoch: 471 Difficulty: 8.59G 


stop at this

**Comment by albanosp** [comment]
Same here.... been trying for hours... nothing works...

![LOLminer_Error](https://user-images.githubusercontent.com/24868343/152666406-c04bf015-457b-46f1-8e3b-0ced05293815.jpg)

**Comment by pabloguns** [comment]
solved, you have to change the pool. I leave the link to mine dual in windows.

https://ton-pool.com/

Server= https://next.ton-pool.com

**Reply by eaamgh** [reply]
Not sure it's a pool-related thing. May be. I have it running on both Windows and HiveOS. It's been running perfectly fine on HiveOS, connected to WhalesTon, for almost 24h. I'm getting this issues on Windows only. What's really weird is if you switch back to ETH-only mining, leave it mining for a couple of minutes, and then to DUALTON, it will start working again for some time. 
Ton-Pool has a 5% fee. Too high IMHO. WhalesTon has no fee.

**Comment by krissboot** [comment]
Changing the pool only made lolminer work longer for before the crash problem occured again.
LOLminer need an update asap.

**Reply by pabloguns** [reply]
I have been working in dual on Windows 10 with lolminer for 14 hours, only changing the pool, I am sure it is a problem of the pool and not of the miner

**Reply by eaamgh** [reply]
I've had it mining on tonwhales for about the same time before it started crashing. I don't thing it's the pool. Or at least not just the pool. It runs perfectly fine on HiveOS, connected to the same pool. Should this be a pool-related problem, it should crash on Hive as well. Only Windows miners are showing this unstable behavior, at least for me.

**Reply by pabloguns** [reply]
yes, apparently this happens only on Windows. I changed the pool, and I have it running for more than 20 hours. From what I could read from other users on youtube, the problem is the pool, and it only exists on Windows. In hiveos it works fine. The problem is that I have two 3060V1, and in hiveos I lose a lot of hashrate mining ETH.

**Comment by eaamgh** [comment]
Yep. Fixed on 1.44. Running stable for over 2h now

**Reply by pabloguns** [reply]
Thanks, I'm updating and testing, apparently the problem is when the "connection" with the TON pool in windows is lost and the crash occurs that leaves us without mining ETH. Version 1.44 apparently solves this problem. I hope so, I'm trying it. Greetings

**Comment by pabloguns** [comment]
I just did the test with version 1.44 of LOLMINER, I disconnected the network cable in Windows 10 dual mining, ETH+TON (https://ton-pool.com/) and the miner did not finish, I am waiting for the internet to come back . It would only be necessary to prove what happens if only the TON poll falls, would it continue mining ETH? I don't know how to test. But that would be the final test to be able to mine in peace.

**Comment by pabloguns** [comment]
Apparently the problem arises when more than 2 RIGS (equipment) (ip) are connected to the TON pool (whatever it is) to the same TON Wallet, I am using version 1.44 of lolminer. With two teams I had no problems, when adding a third, the first crashes and the second and third remain. Too bad it's not stable yet. Be careful, these words are only the experiences of a "layman" I hope that together we can solve it.
